### PR TITLE
Jazzer Pro

### PR DESCRIPTION
**Pull request by kyakdan**
Make the latest version of Jazzer available in the public repository under a new license, allowing for use with open-source projects, academic research, and automated analysis for all projects in OSS-Fuzz.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +143,17 @@ Information on building and testing Jazzer for development can be found in [CONT
...
 
+## License
+By using Jazzer, you agree to the Code Intelligence [Terms & Conditions](LICENSE).
+
+Jazzer can only be used for code bases that are released under an OSI-approved open source license, 
+or to perform academic research. It cannot be used to fuzz projects for or during automated analysis, 
+continuous integration or continuous delivery, whether as part of normal software engineering processes or otherwise. 
+For these use cases, get in contact with us via [sales@code-intelligence.com](mailto:sales@code-intelligence.com).
+
+You can find the latest version released under the Apache 2 license [here](https://github.com/CodeIntelligenceTesting/jazzer/tree/d2cbfdcfc5363593f36cd972b849cc3ab070c90a).
```
Comment (MarkusZoppelt) [inline README.md:124]: This is a permalink which is contradictory to "latest version", no?

File: LICENSE-JAZZER.txt
Comment (MarkusZoppelt) [inline LICENSE-JAZZER.txt:1]: Having this file, which is only a symlink to the LICENSE and having it referenced in every source file makes absolutely no sense to me and actually makes things more confusing tbh.

**Review by MarkusZoppelt — APPROVED** [review]
The branch looks good to me.

I confirmed that

```terminal
bazel build //:jazzer_release
```
and
```terminal
bazel test //...
```

are working on a fresh install and tested out some projects.

File: LICENSE [hunk 2]
```diff
@@ -1,202 +1,180 @@
...
+    standalone basis or combined, incorporated or integrated with other software
+    or services) for others to use, or transfer the Software to any third party.
+
+Code Intelligence reserves all rights not expressly granted in these Terms. 
+
+Open Source Software
+--------------------
+
+The Software may include components licensed under open source
+software licenses. Any such licenses are included in the "NOTICE"-file that
```
Comment (fmeum) [inline LICENSE:98]: The `NOTICE` file is missing from this PR.

**Comment by fmeum** [comment]
If you are interested, I could send a follow-up PR that wires up the LICENSE/NOTICE file with Bazel's way of tying a project to license information. That would make license restrictions more visible to anyone using Jazzer.

**Comment by kyakdan** [comment]
> If you are interested, I could send a follow-up PR that wires up the LICENSE/NOTICE file with Bazel's way of tying a project to license information. That would make license restrictions more visible to anyone using Jazzer.

@fmeum That would be awesome! Could you push this change to this branch directly or create a PR on this branch so that I can integrate it with the first commit that changes the license?

**Comment by fmeum** [comment]
@kyakdan I sent #907.

File: LICENSE-JAZZER.txt
Comment (kyakdan) [reply LICENSE-JAZZER.txt:1]: After checking again, this was the clear advice we got to make the license changes clear!

File: LICENSE (hunk 2 above)
Comment (kyakdan) [reply LICENSE:98]: We added the line `This file also contains code licensed under Apache2 license.` to the headers of source files containing contributions from non-CI employees. This should be enough for now. Will add a NOTICE file later in a separate PR.

**Pull request merged**